extensions.json

```diff
@@ -430,7 +420,8 @@
     "repository": "https://github.com/halcyon-tech/code-for-ibmi"
   },
   "halcyontechltd.git-client-ibmi": {
-    "repository": "https://github.com/halcyon-tech/git-client-ibmi"
+    "repository": "https://github.com/halcyon-tech/git-client-ibmi",
+    "prepublish": "[ $VERSION = '0.1.0' ] && mv LICENCE LICENSE || true"
```

Comment:
Does `mv LICENSE LICENSE` do something here?

Reply:
Yes - the file is incorrectly called `LICENCE`, thus `ovsx` is unable to detect the license.
Renaming the file to `LICENSE` fixes the issue.

Reply:
Looks like the author has setup OpenVSX publishing, and it just failing because of the license: https://github.com/halcyon-tech/git-client-ibmi/blob/master/.github/workflows/ci.yaml.